variables.tf

```diff
@@ -1,17 +1,17 @@
-variable "tenant" {
+variable "control_plane" {
```

Comment:
Inverting positions for variables are just creating unnecessary changes. Is there a reason I did not see?

Reply:
Not really, I just thought that It would be more organized if ordered this way. I've reverted this change.